src/psqlgraph/poly.py

```diff
@@ -0,0 +1,52 @@
+"""A module for poly edges/nodes which generate an entity based on a label."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from psqlgraph import edge, node
+
+
+def PolyEdge(
+    *,
+    label: str,
+    src_id: str | None = None,
+    dst_id: str | None = None,
+    acl: list[str] | None = None,
+    system_annotations: Mapping[str, Any] | None = None,
+    properties: Mapping[str, Any] | None = None,
```

Comment:
I will recommend keeping changes that affect downstream projects to as small as possible. psqlgraph is open source, not sure how we communicate such changes to people that may be using it for their projects already. For this particular function, compared to the potential downstream effort, I don't see much value in re-ordering the parameters and changing it to keyword only.

Reply:
Is the concern with the label not being optional? One way we can communicate would be tagging it as a major version change and specifying this in the release notes.

Reply:
Think this was more or less discussed at the team meeting and it was agreed that this is the best time to break bad interfaces. Also did not realize I hadn't added a doc string. So that is there now and should further add clarity of how to use these functions.